docs/content/en/docs/concepts/architecture/components/metrics-operator/_index.md

```diff
@@ -33,22 +46,65 @@ style Metrics-Adapter fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
+provided by the selected metric provider.
+Each `KeptnMetric` is identified by `name`
+and is associated with an instance of an observability platform
+that is defined in a
+[KeptnMetricsProvider](../../../../yaml-crd-ref/metricsprovider.md)
+resource.
 
-**Metrics controller** is used to fetch metrics from a SLI provider.
-The controller reconciles a [`KeptnMetric`](../../../../yaml-crd-ref/metric.md) CR and
-updates its status with the metric value provided by the selected SLI provider.
 The steps in which the controller fetches metrics are given below:
 
-* It first fetches the `KeptnMetric` object to reconcile.
-* If the object is not found, it returns and lets Kubernetes handle deleting all associated resources.
-* If the object is found, the code checks that if the metric has been updated within the configured
-interval which is defined in the `Spec.FetchIntervalSeconds`.
-If not, then it skips reconciling and requeues the request for later.
-* If the metric should be reconciled, it fetches the provider defined in the `Spec.Provider.Name` field.
-* If the provider is not found, it returns and requeues the request for later.
-* If the provider is found, it loads the provider and evaluates the query defined in the `Spec.Query` field.
-* If the evaluation is succesful, it stores the fetched value in the status of the `KeptnMetric` object.
+1. Fetch the `KeptnMetric` object to be reconciled.
+   * If the object is not found,
+     it returns and lets Kubernetes handle deleting all associated resources.
+1. If the object is found, the code checks
+   whether the metric has been updated within the interval
+   that is defined in the `spec.fetchintervalseconds` field.
+   * If not, it skips reconciling and requeues the request for later.
+1. If the metric should be reconciled,
+   it fetches the provider defined in the `spec.provider.name` field.
+   * If the provider is not found,
+     it returns and requeues the request for later.
+1. If the provider is found,
+   it loads the provider and evaluates the query
+   defined in the `spec.query` field.
+1. If the evaluation is successful,
+   it stores the fetched value
+   in the metric status field of the `KeptnMetric` object.
+1. If the evaluation fails,
+   the error and reason is also provided in the
+   [MetricStatus](../../../../crd-ref/metrics/v1alpha3/#keptnmetricstatus)
+   resource.
+   The error is described in human-readable language
+   and as raw data to help identify the source of the problem
+   (such as a forbidden code).
+
+   The `MetricStatus` resource includes the following information:
+
+   ```yaml
+   properties:
+     errMsg:
+       description: provides error details if the query could not be evaluated
+       type: string
+     lastUpdated:
+       description: time when the status data was last updated
+       format: date-time
+       type: string
+     rawValue:
+       description: shows the resulting value, in raw format
+       format: byte
+       type: string
```

Comment:
`value` field is missing here in the properties list

Reply:
I added it.  Not sure what to use as the value of format -- is it "byte" to match rawValue or is it not specified as for errMsg?  Also, I notice that the properties listed here are in a different order than what shows in the API ref.  Should I reorder this list to match the API ref or does it not matter?

Reply:
I added `value` field but wasn't sure what to use for `format` -- should it be `byte` like `rawValue` or is it not specified like `errMsg`?  Also I notice that the `properties` here are listed in a different order than in the API ref.  Should I reorder this list to match the API ref or does it not matter?

Reply:
done